src/main/java/fi/otavanopisto/kuntaapi/server/rest/CodesApiImpl.java

```diff
@@ -0,0 +1,109 @@
+package fi.otavanopisto.kuntaapi.server.rest;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import javax.ejb.Stateful;
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.Request;
+import javax.ws.rs.core.Response;
+
+import org.apache.commons.lang3.EnumUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import fi.metatavu.kuntaapi.server.rest.CodesApi;
+import fi.metatavu.kuntaapi.server.rest.CodesApi;
+import fi.metatavu.kuntaapi.server.rest.model.Code;
+import fi.otavanopisto.kuntaapi.server.controllers.CodeController;
+import fi.otavanopisto.kuntaapi.server.controllers.HttpCacheController;
+import fi.otavanopisto.kuntaapi.server.controllers.CodeController;
+import fi.otavanopisto.kuntaapi.server.id.OrganizationId;
+import fi.otavanopisto.kuntaapi.server.id.CodeId;
+import fi.otavanopisto.kuntaapi.server.integrations.KuntaApiConsts;
+import fi.otavanopisto.kuntaapi.server.integrations.CodeSortBy;
+import fi.otavanopisto.kuntaapi.server.integrations.CodeSortBy;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this duplicated import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)